### PR TITLE
Adding unicorn-binance-websocket-api

### PR DESCRIPTION
**Pull request by oliver-zehentleitner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by oliver-zehentleitner** [comment]
@conda-forge-admin, please ping team

File: recipes/unicorn-binance-websocket-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - colorama
+    - flask
+    - flask-restful
+    - psutil
+    - python >=3.7
+    - requests
+    - ujson
+    - unicorn-fy
+    - websocket-client
+    - websockets ==10.2
```
Comment (BastianZim) [inline recipes/unicorn-binance-websocket-api/meta.yaml:32]: Are you sure all of this is needed?

File: recipes/unicorn-binance-websocket-api/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+test:
+  imports:
+    - tools
```
Comment (BastianZim) [inline recipes/unicorn-binance-websocket-api/meta.yaml:48]: This means that you're installing a separate package called tools. Is that on purpose?

File: recipes/unicorn-binance-websocket-api/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-websocket-api
+  summary: An unofficial Python API to use the Binance Websocket API`s (com+testnet, com-margin+testnet, com-isolated_margin+testnet, com-futures+testnet, jersey, us, jex, dex/chain+testnet) in a easy, fast, flexible, robust and fully-featured way.
+  license: MIT
+  license_file: LICENSE
+  dev_url: https://www.lucit.tech
```
Comment (BastianZim) [inline recipes/unicorn-binance-websocket-api/meta.yaml:60]: Same as in the other PR.

File: recipes/unicorn-binance-websocket-api/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  doc_url: https://unicorn-binance-websocket-api.docs.lucit.tech/
+  doc_source_url: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-websocket-api/blob/master/README.md
```
Comment (BastianZim) [inline recipes/unicorn-binance-websocket-api/meta.yaml:62]: I would remove this as well.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Otherwise, good to go.

File: recipes/unicorn-binance-websocket-api/meta.yaml (hunk 1 above)
Comment (oliver-zehentleitner) [reply recipes/unicorn-binance-websocket-api/meta.yaml:32]: yes, i am the author and they are all used :)

File: recipes/unicorn-binance-websocket-api/meta.yaml (hunk 2 above)
Comment (oliver-zehentleitner) [reply recipes/unicorn-binance-websocket-api/meta.yaml:48]: no, thats wrong, thanks

File: recipes/unicorn-binance-websocket-api/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/unicorn-binance-websocket-api/meta.yaml:32]: Yes but are they needed during building? It seems like only pip and python is required and the rest only in run.

Ref: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-websocket-api/blob/ccbe1a67f00d444b4ea709e5ccaff76db4f811fa/setup.py#L56

Comment (BastianZim) [reply recipes/unicorn-binance-websocket-api/meta.yaml]: Same here: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-websocket-api/blob/ccbe1a67f00d444b4ea709e5ccaff76db4f811fa/setup.py#L37

You're also importing the package when building it so the deps need to stay in host for now.

File: recipes/unicorn-binance-websocket-api/meta.yaml [hunk 5]
```diff
@@ -45,7 +34,6 @@ requirements:
 
 test:
   imports:
-    - tools
```
Comment (BastianZim) [inline recipes/unicorn-binance-websocket-api/meta.yaml]: This is just removing it from the tests but the issue will still persist. In the end, you'll need to update this upstream.

**Review by BastianZim — APPROVED** [review]

**Pull request merged**